enterprise/server/remote_execution/runner/runner_test.go

```diff
@@ -339,7 +339,14 @@ func TestRunnerPool_Shutdown_RunnersReturnRetriableOrNilError(t *testing.T) {
 	// Run 100 trials where we create a pool that runs 50 tasks using runner
 	// recycling, shutting down the pool after roughly half of the tasks have been
 	// started.
-	for i := 0; i < 100; i++ {
+	//
+	// On macOS, we run 30 trials because the Intel macOS CI is slow and we want to
+	// avoid timeouts.
+	trialCount := 100
+	if runtime.GOOS == "darwin" {
+		trialCount = 30
+	}
+	for i := 0; i < trialCount; i++ {
```

Comment:
For simplicity, we can set this to 30 on all platforms. This test is already a bit overkill even on Linux

```suggestion
	for i := 0; i < 30; i++ {
```